README.md

```diff
@@ -10,9 +10,9 @@
 
 ## Overview
 
-Reactron is a tiny Electron project configured to work with React as the frontend. The project has just the necessary to start a new app.
+Reactron is a tiny Electron project built to handle with React as frontend. The project has just the necessary to start a new app.
```

Comment:
Please keep the old message

Reply:
_The messages is equal the old I just fix some grammar points/words._